test/common_nn.py

```diff
@@ -1664,6 +1664,19 @@ def multimarginloss_weights_no_reduce_test():
         fullname='FractionalMaxPool2d_size',
         test_cuda=False,
     ),
+    dict(
+        constructor=lambda: nn.FractionalMaxPool3d(
+            2, output_ratio=0.5, _random_samples=torch.DoubleTensor(2, 4, 3).uniform_()),
+        input_size=(2, 4, 5, 5, 5),
+        fullname='FractionalMaxPool3d_ratio',
+    ),
+    dict(
+        constructor=lambda: nn.FractionalMaxPool3d((2, 2, 2), output_size=(
+            4, 4, 4), _random_samples=torch.DoubleTensor(2, 4, 3).uniform_()),
```

Comment:
First off, it's awesome that you added these tests, thanks a lot :) (I didn't even realize that this function wasn't exposed from Python when I originally bootcamped this task). What would be really good is to add some tests which test for asymmetric kernel size. Something like in https://github.com/pytorch/pytorch/pull/15823

Reply:
The asymmetric test is added as suggested.